src/errors/error.es6.js

```diff
@@ -0,0 +1,15 @@
+export default class ErrorClass {
```

Comment:
would there be any value to using `util.inherits` to inherit the prototype from `Error`?